### ISSUE TITLE
Android build fails under side-by-side NDK (r19+)

### ISSUE THREAD

**Issue by solodon4** [microsoft/cpprestsdk#1629]
NDK r19 is now considered obsolete and unsupported by Android. That was the last time non side-by-side NDK was supported, which is the only way cpprestsdk currently builds on Android. I didn't have any luck building cpprestsdk with more recent NDKs. Has anyone had any experience building cpprestsdk with side-by-side NDK and could share some details? Thank you!

**Comment by barcharcraz** [comment]
I have it on my list to update the android CI for a newer NDK.

**Comment by solodon4** [comment]
Thank you @barcharcraz! I also once sent a pull request to [enable GitHub Actions under iOS/Android](https://github.com/microsoft/cpprestsdk/pull/1435) but it was never merged in. That one was still for non-side-by-side NDK, but it would be great if you guys enable it since GitHub action workflows travel with forks and people trying to contribute to cpprestsdk would know immediately when they broke your two least tested scenarios.

**Comment by barcharcraz** [comment]
Yeah, I'd prefer to just continue to use azure pipelines, since we have access to more resources there. The CI is not enabled for pull requests, so maybe just enabling it would be better.
